src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1862,7 +1862,7 @@
                       Grid.Row="2"
                       Grid.RowSpan="2"
                       Height="Auto"
-                      Width="0"
+                      Width="3"
```

Comment:
where is this coming from?

Reply:
I thought we applied this fix already before.. Was it only added to 2.3 branch?

Reply:
Actually, this change is in master but I did not reset my head when I was starting this PR. So it came up. https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Core/DynamoView.xaml#L1865